deploy/docker/create-docker-context-for-node-components.js

```diff
@@ -1,345 +0,0 @@
-#!/usr/bin/env node
```

Comment:
@steve9164 is removing this file why the deploy job fails in the main branch of the TerriaJS repository?